src/ObjectSerializer.php

```diff
@@ -28,81 +29,87 @@
-
 /**
- * ObjectSerializer Class Doc Comment
+ * ObjectSerializer Class Doc Comment.
  *
  * @category Class
- * @package  Fingerprint\ServerAPI
+ *
  * @author   Swagger Codegen team
- * @link     https://github.com/swagger-api/swagger-codegen
+ *
+ * @see     https://github.com/swagger-api/swagger-codegen
  */
 class ObjectSerializer
 {
     /**
-     * Serialize data
+     * Serialize data.
      *
      * @param mixed  $data   the data to serialize
      * @param string $format the format of the Swagger type of the data
      *
-     * @return string|object serialized form of $data
+     * @return object|string serialized form of $data
      */
     public static function sanitizeForSerialization($data, $format = null)
     {
         if (is_scalar($data) || null === $data) {
             return $data;
-        } elseif ($data instanceof \DateTime) {
-            return ($format === 'date') ? $data->format('Y-m-d') : $data->format(\DateTime::ATOM);
-        } elseif (is_array($data)) {
+        }
+        if ($data instanceof \DateTime) {
+            return ('date' === $format) ? $data->format('Y-m-d') : $data->format(\DateTime::ATOM);
+        }
+        if (is_array($data)) {
             foreach ($data as $property => $value) {
                 $data[$property] = self::sanitizeForSerialization($value);
             }
+
             return $data;
-        } elseif ($data instanceof \stdClass) {
+        }
+        if ($data instanceof \stdClass) {
             foreach ($data as $property => $value) {
-                $data->$property = self::sanitizeForSerialization($value);
+                $data->{$property} = self::sanitizeForSerialization($value);
             }
+
             return $data;
-        } elseif (is_object($data)) {
+        }
+        if (is_object($data)) {
             $values = [];
             $formats = $data::swaggerFormats();
             foreach ($data::swaggerTypes() as $property => $swaggerType) {
                 $getter = $data::getters()[$property];
-                $value = $data->$getter();
-                if ($value !== null
+                $value = $data->{$getter}();
+                if (null !== $value
                     && !in_array($swaggerType, ['DateTime', 'bool', 'boolean', 'byte', 'double', 'float', 'int', 'integer', 'mixed', 'number', 'object', 'string', 'void'], true)
                     && method_exists($swaggerType, 'getAllowableEnumValues')
                     && !in_array($value, $swaggerType::getAllowableEnumValues())) {
                     $imploded = implode("', '", $swaggerType::getAllowableEnumValues());
-                    throw new InvalidArgumentException("Invalid value for enum '$swaggerType', must be one of: '$imploded'");
+
+                    throw new \InvalidArgumentException("Invalid value for enum '{$swaggerType}', must be one of: '{$imploded}'");
                 }
-                if ($value !== null) {
+                if (null !== $value) {
```

Comment:
lol 😆
nit: I understand that in PHP world people sometimes do this kind of comparisons, but TBH sometimes it's hard to wrap your head around them. Consider updating php-cs to disallow this kind of comparisons, but it's up to you.

Reply:
I'll leave this to refactor PR